bazel/compat.bzl

```diff
@@ -26,3 +26,11 @@ LINUX_ONLY = select({
     "@platforms//os:linux": [],
     "//conditions:default": ["@platforms//:incompatible"],
 })
+
+MACOS_MULTI_PLATFORM = select({
```

Comment:
This enables multi platform builds for macOS but is also evaluated for Windows and Linux, right? If so, should we remove the `MACOS` prefix?

Reply:
Good point. I renamed it to `MULTI_PLATFORM_ON_MACOS` to make it clear that it only has an effect on macOS but can still be used on every OS.

Reply:
Reconsidered and removed the suffix altogether as you suggested - we may want to add Windows ARM support with this mechanism.